edb/pgsql/delta.py

```diff
@@ -2019,52 +2044,157 @@ def constraint_is_effective(
                 return not subject.get_abstract(schema)
         raise NotImplementedError(subject)
 
-    @classmethod
-    def fixup_base_constraint_triggers(
-        cls, constraint, orig_schema, schema, context, span=None, *, is_delete
+    def schedule_relatives_constraint_trigger_update(
+        self,
+        constraint: s_constr.Constraint,
+        orig_schema: s_schema.Schema,
+        curr_schema: s_schema.Schema,
+        context: sd.CommandContext,
     ):
-        base_schema = orig_schema if is_delete else schema
+        # Find all origins whose relationship with the constraint has changed.
+        orig_origins: dict[uuid.UUID, s_constr.Constraint] = {}
```

Comment:
My inclination would be to use the `Object` as the key instead of the uuid. It doesn't really matter, but feels cleaner to me. (Behavior is the same.)